services/madoc-ts/src/routes/projects/delete-project-summary.ts

```diff
@@ -0,0 +1,67 @@
+import { sql } from 'slonik';
+import { api } from '../../gateway/api.server';
+import { RouteMiddleware } from '../../types/route-middleware';
+import { userWithScope } from '../../utility/user-with-scope';
+
+export const deleteProjectSummary: RouteMiddleware<{ id: number }> = async context => {
+  const {siteId} = userWithScope(context, ['site.admin']);
+  const projectId = context.params.id;
+
+  const siteApi = api.asUser({siteId});
+
+  // Fact checking stage.
+  const { site_count } = await context.connection.one(
+    // This will let us know if the manifest appears on any other sites.
+    // If === 1 then we can safely delete underlying resource.
+    sql<{ site_count: number; manifest_count: number }>`
+      select COUNT(distinct site_id) as site_count from iiif_derived_resource where resource_id=${projectId}
+    `
+  );
+
+  const { collection_count } = await context.connection.one(
+    sql<{ collection_count: number }>`
+      select COUNT(*) as collection_count from iiif_derived_resource
+        where
+          id in (select item_id from iiif_derived_resource_items where resource_id = &{projectId})
```

Comment:
The project contains a `collection_id` on its table. This is what the `resource_id` should match to find the manifests/collections. Easiest thing might be to fetch from the API: `const project = await siteApi.getProject(projectId)` and then `project.collection_id` is what you need

Reply:
Also in template literals you need to use `${}` instead of `&{}`

Reply:
> The project contains a `collection_id` on its table. This is what the `resource_id` should match to find the manifests/collections. Easiest thing might be to fetch from the API: `const project = await siteApi.getProject(projectId)` and then `project.collection_id` is what you need

Ah yeah, good catch. In my local environment the project ID was always the same as it's collection ID, but that probably isn't a fair assumption in all cases.

> Also in template literals you need to use ${} instead of &{}

🤦 Oops.